build/gulpfile.hygiene.js

```diff
@@ -199,7 +199,7 @@ gulp.task('tslint', () => {
 			.pipe(filter(tslintExtensionsFilter))
 			.pipe(gulptslint.default({ rulesDirectory: 'build/lib/tslint' }))
 			.pipe(gulptslint.default.report({ emitError: true }))
-	]);
+	]).pipe(es.through());
```

Comment:
Added this since the gulp task was failing when no errors were being thrown. It still seems to fail correctly when actual errors occur.